students/C_Farris/session01/break_me.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+#Author: Carol Farris
+#File Name: break_me.py
+#Content: Write four simple python functions that throw errors
+
+
+#The function below will throw a name error if supplied an undefined value
+def cause_name_error(val):
+    try:
+        print(val)
+    except NameError:
```

Comment:
did you actually get this to raise? right now, "val" is defined?

And we're not expecting you to catch these error yet, just wanted you to experience seeing them.